src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -493,7 +500,7 @@ public void testMultiColumnIngest() throws Exception {
         Assert.assertEquals(3.14, result.getFloat("F"), 0.0001);
         Assert.assertEquals(1.1, result.getFloat("TINYFLOAT"), 0.001);
         Assert.assertEquals("{\n" + "  \"e\": 2.7\n" + "}", result.getString("VAR"));
-        Assert.assertEquals(timestamp * 1000, result.getTimestamp("T").getTime());
+        Assert.assertEquals(timestamp * 1000, result.getTimestamp("T", cal).getTime());
         Assert.assertEquals(-923, TimeUnit.MILLISECONDS.toDays(result.getDate("D").getTime()));
```

Comment:
@sfc-gh-mnaides looks like we're still failing on the date comparison, any idea why?

Reply:
I think you need to add cal to the getDate() call as well

Reply:
ok, I think the issue is the Date returned has a timezone based on the session setting, so I need to set to UTC in the session